src/main/java/plugily/projects/buildbattle/handlers/menu/registry/particles/ParticleItem.java

```diff
@@ -41,10 +39,6 @@ public void setPermission(String permission) {
     this.permission = permission;
   }
 
-  public String getDisplayName() {
-    return ComplementAccessor.getComplement().getDisplayName(itemStack.getItemMeta());
-  }
```

Comment:
This method isn't used anywhere so I removed this to reduce the byte codes, actually this displayName must be avoided.